.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Questions and help
+    url: https://discourse.metabase.com
+    about: For help with installation and setup, information on how specific features work, or general questions about Metabase, please use our discussion forum.
```

Comment:
This will now render as an item list in the issue picker with the ability to directly open our Discourse.
![image](https://user-images.githubusercontent.com/31325167/233336128-69acbab6-38c9-4f36-b38a-1460d50cd880.png)